js/src/setup-ads/ads-stepper/setup-accounts/google-ads-account-card/authorize-ads.js

```diff
@@ -13,6 +13,11 @@ import AppButton from '.~/components/app-button';
 
 const pageName = 'setup-ads';
 
+/**
+ * @param {Object} props React props
+ * @param {string} props.additionalScopeEmail
```

Comment:
Not sure what is this `additionalScopeEmail`  for, but the description is missing.

Reply:
I was not sure either. Just added doc block to put `@fires` and linter complained about not mentioned params.